public/stylesheets/faq.css

```diff
@@ -25,8 +25,9 @@ figure figcaption {
 }
 
 .faq .faq-item h2 {
-    margin-top: 0;
+    margin-top: -70px;
```

Comment:
Is this the recommended way of solving this issue? Did you decide the constant based on height of the navbar or something else? Essentially, I am trying to understand whether the same values can be used elsewhere with the same values (e.g. #800) or is it something else?